### DISCUSSION TITLE
GitHub Actions file list not matching with current.

### DISCUSSION THREAD

**Discussion by behoyh** [community/community#77385] [Actions]
### Select Topic Area

Bug

### Body

Hello,

I am trying to pass a file as an argument to a command step in a GitHub Action.

Currently the files are not updating (added a new file and overwrote an existing file) as evidenced by running `ls`. The Action is triggered on release through the UI.

Is there anyway to update these files to match the repo?

**Comment by Varsha-JJ** [comment]
Working with GitHub Actions and you want to pass a file as an argument to a command in a workflow

**Reply by behoyh** [reply]
Yes
